### DISCUSSION TITLE
検索ボックスを表示させた際にカーソルが挿入されてほしい

### DISCUSSION THREAD

**Discussion by yarakos95** [increments/qiita-discussions#479] [要望・アイデア・不具合]
現在、Qiita の上部にある検索ボタン :mag:  を押下すると検索ボックスが表示されますが、カーソルは挿入されません。

![searchClick](https://github.com/increments/qiita-discussions/assets/75878492/49ac28fb-bf80-4f14-ace0-946aa74c335c)

Qiita 内で検索する際に 2 つの操作をする必要があり少し面倒です。検索ボックスを表示させた時点でカーソルが挿入されていてほしいです。

**Comment by tanaka-asami415** [comment]
@Yarakashi-Kikohshi 
ご意見ありがとうございます🙏
こちらDiscussions以外でも同様のご意見をいただいているため、対応します！

**Comment by kyntk** [comment]
@Yarakashi-Kikohshi お待たせしました！
検索ボックスを表示させるタイミングでFocusがあたるように変更しました！

**Reply by yarakos95** [reply]
検索ボックスを表示させたタイミングでカーソルが挿入される様子を確認しました！
対応ありがとうございます 🎉
